src/components/Login/LoginForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { SelectOption } from '@/types/types'
+
+export interface FormProps {
+  id: string
+  label: string
+  type: string
+  isNumPad: boolean
+  placeholder: string
+  isRequired: boolean
+  selectOptions?: SelectOption[]
+  onChange?: (
+    event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>
+  ) => void
+}
+
+function Form(props: FormProps) {
```

Comment:
This is a very confusing name, it returns an input field not an entire form.

Reply:
Given there's only a couple of input fields for the phone login method, it might be simpler to add them directly to the form rather than abstract out into a builder component like you have here.